mobile/lib/modules/asset_viewer/views/video_viewer_page.dart

```diff
@@ -59,13 +59,15 @@ class VideoViewerPage extends HookConsumerWidget {
 
     return Stack(
       children: [
-        VideoThumbnailPlayer(
-          url: videoUrl,
-          jwtToken: Store.get(StoreKey.accessToken),
-          isMotionVideo: isMotionVideo,
-          onVideoEnded: onVideoEnded,
-          onPaused: onPaused,
-          onPlaying: onPlaying,
+        SafeArea(
```

Comment:
You may not want this safe area here. You typically don't mind when a video paints the entire viewpoints. It gives a more immersive view. I haven't tried to see what it does yet.